sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatThreadProperties.java

```diff
@@ -122,6 +130,46 @@ public ChatThreadProperties setCreatedBy(CommunicationIdentifier createdBy) {
         return this;
     }
 
+    /**
+     * Get the metadata property: Property bag of chat thread metadata key-value pairs.
+     *
+     * @return the metadata map.
+     */
+    public Map<String, String> getMetadata() {
+        return this.metadata;
+    }
+
+    /**
+     * Set the metadata property: Property bag of chat thread metadata key-value pairs.
+     *
+     * @param metadata the metadata map to set.
+     * @return the ChatThreadProperties object itself.
+     */
+    public ChatThreadProperties setMetadata(Map<String, String> metadata) {
+        this.metadata = metadata;
+        return this;
+    }
+
+    /**
+     * Get the retentionPolicy property: Thread retention policy.
+     *
+     * @return the retentionPolicy value.
```

Comment:
Same goes for keeping "value" at the end of setters/getters. In general, I'd avoid copying generated Javadoc word for word.